src/NuGet.ApplicationInsights.Owin/ApplicationInsightsMiddleware.cs

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+using System.Threading.Tasks;
+using Microsoft.ApplicationInsights;
+using Microsoft.ApplicationInsights.DataContracts;
+using Microsoft.ApplicationInsights.Extensibility;
+using Microsoft.Owin;
+
+namespace NuGet.ApplicationInsights.Owin
+{
+    public class ApplicationInsightsMiddleware
+        : OwinMiddleware
+    {
+        public const string OwinRequestIdKey = "owin.RequestId";
+
+        private readonly TelemetryClient _telemetryClient;
+
+        public ApplicationInsightsMiddleware(OwinMiddleware next)
+            : this(next, null)
+        {
+        }
+
+        public ApplicationInsightsMiddleware(OwinMiddleware next, TelemetryConfiguration telemetryConfiguration) 
```

Comment:
Minor: Can we enable the scope for telemetry client as opposed to telemetryconfiguration? I don't see using telemetryConfiguration anywhere. Why not just get telemetry client?